examples/ExecutionPools/AutoPart/test_plan.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""
+This example is to demonstrate auto-part feature in a process pool.
+"""
+
+import sys
+import typing
+from testplan import test_plan
+from testplan import Task
+from testplan.runners.pools.process import ProcessPool
+from testplan.common.utils.testing import calculate_pool_size_by_tasks
+from testplan.report.testing.styles import Style, StyleEnum
+
+OUTPUT_STYLE = Style(StyleEnum.ASSERTION_DETAIL, StyleEnum.ASSERTION_DETAIL)
+
+
+# Set the runtime limitation for auto-part tasks
```

Comment:
The auto_part_runtime_limit argument instructs testplan to split parts="auto" Multitest into optimal number of parts so that the runtime of each part is not more than the limit.